src/gd_webp.c

```
@@ -291,7 +291,7 @@ BGD_DECLARE(void) gdImageWebpEx (gdImagePtr im, FILE * outFile, int quality)
 	if (out == NULL) {
 		return;
 	}
-	_gdImageWebpCtx(im, out, quality);
+	gdImageWebpCtx(im, out, quality);
```

this is incorrect ... it should be using `_gdImageWebpCtx`

---

I figured `gdImageWebpCtx()` was superior because it returns `void`, instead of returning an `int` that gets thrown away. Apparently not?

---

`gdImageWebpCtx` is just going to call `_gdImageWebpCtx`, so it's only adding overhead to call `gdImageWebpCtx` ?

it's also about using the internal interfaces than having to go through the exported symbols which require a PLT entry

---

got it!

---

looks like these changes are still in here when they shouldn't be

---

Sorry, this one dropped off the map. Reverted.